tests/components/vilfo/test_config_flow.py

```diff
@@ -0,0 +1,81 @@
+"""Test the Vilfo Router config flow."""
+from unittest.mock import patch
+
+from homeassistant import config_entries, setup
+from homeassistant.components.vilfo.config_flow import CannotConnect, InvalidAuth
+from homeassistant.components.vilfo.const import DOMAIN
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_HOST, CONF_ID
+
+from tests.common import mock_coro
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.vilfo.config_flow.validate_input",
+        return_value=mock_coro({"title": "Test Title", CONF_ID: "testadmin.vilfo.com"}),
+    ), patch(
+        "homeassistant.components.vilfo.async_setup", return_value=mock_coro(True)
+    ) as mock_setup, patch(
+        "homeassistant.components.vilfo.async_setup_entry",
+        return_value=mock_coro(True),
+    ) as mock_setup_entry:
+        result2 = await hass.config_entries.flow.async_configure(
+            result["flow_id"],
+            {CONF_HOST: "testadmin.vilfo.com", CONF_ACCESS_TOKEN: "test-token"},
+        )
+
+    assert result2["type"] == "create_entry"
```

Comment:
`from homeassistant import data_entry_flow`

create_entry --> `data_entry_flow.RESULT_TYPE_CREATE_ENTRY`
form --> `data_entry_flow.RESULT_TYPE_FORM`

Reply:
I noticed now that I missed this one. Will go over the tests again soon and make some updates.

Reply:
Done! ✅